Python-packages/covidcast-py/docs/index.rst

```diff
@@ -31,10 +31,21 @@ To get started, check out :ref:`getting started <getting-started>`.
    notified of package updates, new data sources, corrections, and other
    updates.
 
-.. warning :: If you use data from the COVIDcast Epidata API to power a public
-   product, dashboard, app, or other service, please download the data you need
-   and store it centrally rather than making API requests for every user. Our
-   server resources are limited and cannot support high-volume interactive use.
+.. warning :: By default, this package submits queries to the API
+   anonymously. All the examples in the package documentation are compatible
+   with anonymous use of the API, but `there are some limits on anonymous
+   queries <https://cmu-delphi.github.io/delphi-epidata/api/api_keys.html>`_
+   which do not apply if you use an API key to authenticate your queries. If you
+   regularly or frequently use our system, please consider `registering for a
+   free API key <https://forms.gle/hkBr5SfQgxguAfEt7>`_ even if your usage falls
+   within the anonymous usage limits. API key usage helps us understand who and
+   how others are using our Delphi Epidata API, which may in turn inform our
+   future research, data partnerships, and funding. As we are a research group,
+   our server resources are limited and cannot support high-volume interactive
+   use (with or without an API key). If you use data from the COVIDcast Epidata
+   API to power a public product, dashboard, app, or other service, please
+   download the data you need and store it centrally rather than making API
+   requests for every user.
```

Comment:
Once we have `covidcast.use_api_key` in `signals.rst`, we could edit this to refer to that, so people see in one place (a) how to register and (b) how to use the keys. Currently this message says how to register but not how to actually use the key you get

Reply:
Done; also split off "please don't hammer our server" into a separate paragraph